build.gradle.kts

```diff
@@ -116,6 +118,8 @@ task<JavaExec>("consoleLauncherTest"){
         systemProperty("cucumber.junit-platform.naming-strategy", "long")
          //Hides cucumber ads
         systemProperty("cucumber.publish.quiet", true)
+        // OPTIONAL: Force test execution even if they are up-to-date according to Gradle or use "gradle test --rerun"
+        outputs.upToDateWhen { false }
```

Comment:
what are the implications of this line?

Reply:
do we want that?

Reply:
Yes, because sometimes the tests are not ran and thus giving us a false positive. This ensures that the results are not from cache.